yandex_music/cover.py

```diff
@@ -31,15 +36,15 @@ class Cover(YandexMusicObject):
     """
 
     def __init__(self,
-                 type=None,
-                 uri=None,
-                 items_uri=None,
-                 dir=None,
-                 version=None,
-                 custom=None,
-                 prefix=None,
-                 error=None,
-                 client=None,
+                 type: Optional[str] = None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'type'](https://app.codacy.com/app/MarshalX/yandex-music-api/pullRequest?prid=4731841)